go/teams/loader_test.go

```diff
@@ -520,3 +520,27 @@ func TestLoaderGetImplicitAdminsList(t *testing.T) {
 	t.Logf("U0 sees the 3 implicit admins")
 	assertImpAdmins(tcs[0].G, *subteamID, []keybase1.UserVersion{fus[0].GetUserVersion(), fus[1].GetUserVersion(), fus[2].GetUserVersion()})
 }
+
+func TestLoaderImplicitAdminsCanLoadSubteam(t *testing.T) {
+	fus, tcs, cleanup := setupNTests(t, 2)
+	defer cleanup()
+
+	t.Logf("U0 creates a root team")
+	parentName, _ := createTeam2(*tcs[0])
+
+	t.Logf("U0 creates a subteam")
+	subteamID, err := CreateSubteam(context.TODO(), tcs[0].G, "sub", parentName)
+	require.NoError(t, err)
+
+	t.Logf("U1 can't load the subteam")
+	_, err = tcs[1].G.GetTeamLoader().ImplicitAdmins(context.TODO(), *subteamID)
+	require.Error(t, err, "should not be able to load subteam without implicit admin status")
+
+	t.Logf("U0 adds U1 as an admin in the root team")
+	_, err = AddMember(context.TODO(), tcs[0].G, parentName.String(), fus[1].Username, keybase1.TeamRole_ADMIN)
+	require.NoError(t, err)
+
+	t.Logf("now U1 can load the subteam")
```

Comment:
Yes though now that I think about it, I should probably  have U1 add a new member or something that really requires the shares secret